app/assets/javascripts/application.js

```diff
@@ -33,7 +33,33 @@ $(function() {
 });
 
 function executeAttachinary() {
+  var js_template = [
+    "<ul class='attachinary_images_list'>",
```

Comment:
I never worked with JST, but looking in ther doc seems to be simple http://ricostacruz.com/backbone-patterns/jst-templates.html